### PR TITLE
Typos from marvin.cs.uidaho.edu: S

### PR DESCRIPTION
**Pull request by skangas**
This replaces PR https://github.com/codespell-project/codespell/pull/1968. I've added in several variations myself.

See: http://marvin.cs.uidaho.edu/misspell.html

### CODE REVIEW

**Comment by skangas** [comment]
Hmm, the tests are failing here for some reason. But I don't see why.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32388,8 +32438,12 @@ segmetned->segmented
 segmetns->segments
 segument->segment
 seguoys->segues
+segway->segue
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32441]: Could be a [Segway](https://en.wikipedia.org/wiki/Segway) PT. Move to _rare_?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -32388,8 +32438,12 @@ segmetned->segmented
...
+segwayed->segued
+segwaying->segueing
 seh->she
 seige->siege
+seina->sienna
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32446]: `seine`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -32744,9 +32824,17 @@ sercive->service
 sercived->serviced
 sercives->services
 serciving->servicing
+serebral->cerebral
+serebrally->cerebrally
+sereous->serious
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32829]: `serous`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -32906,7 +33021,8 @@ sharloton->charlatan
 sharraid->charade
 sharraids->charades
 shashes->slashes
-shatow->château
+shatow->château, châteaux,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33024]: `shadow`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -32906,7 +33021,8 @@ sharloton->charlatan
...
+shatows->châteaus, châteauxs,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33025]: Change to 
```
shatows->châteaux, châteaus, shadows,
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -33263,8 +33464,20 @@ singuarity->singularity
 singuarl->singular
 singulat->singular
 singulaties->singularities
+sinic->cynic
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33467]: `sonic`?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -33302,8 +33515,23 @@ sirectories->directories
...
+sirvayls->surveils
+sirynge->syringe
+sirynges->syringes
 sise->size, sisal,
 sisnce->since
+sisser->scissor
+sissered->scissored
+sissering->scissoring
+sissers->scissors
+sist->cyst
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33534]: `sits`, `sift`?

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -33302,8 +33515,23 @@ sirectories->directories
...
+sirvayled->surveiled
+sirvaylence->surveillance
+sirvayles->surveils
+sirvayling->surveiling
+sirvayls->surveils
+sirynge->syringe
+sirynges->syringes
 sise->size, sisal,
 sisnce->since
+sisser->scissor
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33530]: `sissier`, `sister`?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -33302,8 +33515,23 @@ sirectories->directories
...
+sissered->scissored
+sissering->scissoring
+sissers->scissors
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33533]: `sisters`?

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -33323,6 +33551,7 @@ sistemized->systemized
 sistemizes->systemizes
 sistemizing->systemizing
 sistems->systems
+sists->cysts
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33554]: `sifts`, `sits`?

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -33416,6 +33674,11 @@ sligthly->slightly
 sligtly->slightly
 sliped->slipped
 sliseshow->slideshow
+slite->sleight
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33677]: `elite`, `slide`, `site`?

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -33416,6 +33674,11 @@ sligthly->slightly
...
+slooth->sleuth
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:33678]: `sloth`, `sooth`?

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 13]
```diff
@@ -357,6 +357,8 @@ sterilised->sterilized
 steriliser->sterilizer
 sterilises->sterilizes
 sterilising->sterilizing
+storey->story
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:360]: Move to _rare_ (synonym of floor), same for plural.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:32441]: I don't know. It's technically just a brand name and not a proper word, just like the new hip start-up `Frobnicatr` is just a typo for `Frobnicator` and should probably be treated as such. I'd ask projects referring to such brands to fix it locally, in their project ignore file or similar.

WDYT?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:32446]: Added.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33024]: Yup, added.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33025]: Done.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:32829]: Added.

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33467]: Yes, added

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33534]: Added

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33530]: Added.

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33533]: Added.

File: codespell_lib/data/dictionary.txt (hunk 10 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33554]: Added

File: codespell_lib/data/dictionary.txt (hunk 11 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33677]: Added

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:33678]: Added

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt (hunk 13 above)
Comment (skangas) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:360]: Note that this is the `dictionary_en-GB_to_en-US.txt` file: https://en.wikipedia.org/wiki/Storey#cite_note-Cambridge_University_Press_-_Cambridge_Dictionary_-_storey-1

So I think it looks correct?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:33250]: Agreed, let's avoid it. Besides, that brand is less fashionable nowadays.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**